install/fedora-installation-guide.md

```diff
@@ -2,6 +2,14 @@
 
 1. Install the Kata Containers components with the following commands:
 
+   1.1. From Fedora 31 onwards:
+
+   ```bash
+   $ sudo dnf -y install kata-runtime kata-proxy kata-shim
+   ```
+
+   1.2. For Fedora 30 or older:
```

Comment:
Although these instructions are fine for humans, we actually "execute" these documents as scripts in the CI to ensure that they generate a working system.

As such, we need to explain in words that F31 and newer can use the Fedora packages. But we also need logic so that the CI can dtrt. Something like:

<pre>
```bash
$ source /etc/os-release
$ if [ $VERSION_ID -ge 31 ]; then
    sudo dnf -y install kata-runtime
else
  # the existing block of code
  ARCH=$(arch)
  ....
fi
```
</pre>